docs/getting_started.md

```diff
@@ -174,3 +174,9 @@ There are several things you need to check before making your putting your appli
 
 [docker]: https://docs.docker.com/engine/installation/
 [docker-compose]: https://docs.docker.com/compose/install/
+
+## Contributing
+
+We always welcome new contributors of all levels to the project. If you are not confident enough with Ruby or web development you can look for [issues](https://github.com/decidim/decidim/issues) labeled `good first issue` to start contibuting and learning the internals of the project by doing easy jobs.
+
```

Comment:
I'd add something like: 

"You can also find other ways of helping us on our [contribution guide](CONTRIBUTING.md)."